### DISCUSSION TITLE
QuasarRouteTab to be keep active even when in child route

### DISCUSSION THREAD

**Discussion by eevan7a9** [quasarframework/quasar#10541] [General - Components / Directives / etc]
Hello Everyone,

I have a RouteTab that's active in route "/account"

```
    <q-route-tab
          class="full-width"
          align="start"
          active-class="active-route-tab"
          to="/account"
        >
          <div class="full-width>
            ....
            Users Name 
          </div>
    </q-route-tab>
```
but when I navigate to "**/account/profile**"  the active state is gone.

how can I keep the route tab active when in "**/account**" and "**/account/profile**" etc...

this is my **routes.js**

```
    const routes = [
      {
        path: "/",
        component: () => import("layouts/MainLayout.vue"),
        children: [
          {
            path: "",
            redirect: "freeboard",
          },
          {
            path: "account",
            component: () => import("src/pages/Account/Index.vue"),
            name: "account",
          },
          {
            path: "account/:id",
            component: () => import("src/pages/AccountDetails/Index.vue"),
            name: "accountDetails",
          },
        ]
      },
     ....
    ];
```

**Comment by ontwikkelfabriek** [comment]
@eevan7a9 

I think this is a bug / not supposed to happen. Because you're supposed to **need** to set the property  `exact` = `true` to make sure that it will **NOT** match and become active. See:

https://quasar.dev/vue-components/tabs#connecting-to-vue-router
https://router.vuejs.org/api/#exact

Could you post your `routes.js` file that contains the `/account` and `/account/profile` routes?

**Reply by eevan7a9** [reply]
the routes are really simple `/account`  &&  `/account/:id` something like this

**Reply by ontwikkelfabriek** [reply]
@eevan7a9 

I see. You could create an issue in the bug tracker for this.
